configuration/config.js

```diff
@@ -10,7 +10,7 @@ config.host = "0.0.0.0";
 // TCP Port used by the API.
 config.port = "55000";
 // Use HTTP protocol over TLS/SSL. Values: yes, no.
-config.https = "no";
+config.https = "yes";
```

Comment:
If you change this value to `yes`, what happens if the HTTPS can't be set up because the `openssl` is missing?

Reply:
API will be installed with `HTTPS` disabled:
https://github.com/wazuh/wazuh-api/blob/enable-https-by-default-436/install_api.sh#L468-L471